Update Get-CsOnlineUser.md (#10555)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineUser.md

### -Identity
Indicates the Identity of the user account to be retrieved.

> For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are only rolled out in commercial environments at present and will be applicable to the latest TPM versions in GCCH and DOC environments starting March 31, 2023.):

Reviewer: ```suggestion
For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are only rolled out in commercial environments including GCC at present, and will be applicable to the latest TPM versions in GCC High and DoD environments starting March 31, 2023.):
```

> For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are only rolled out in commercial environments including GCC at present, and will be applicable to the latest TPM versions in GCC High and DoD environments starting March 31, 2023.):

Reviewer: March 31 or 15?

Author: it's March 15 now

Reviewer: We need to update it also in line 145

```
Get-CsOnlineUser -Filter {ExternalAccessPolicy -ge "xyz_policy"}
```

> **Updates in Teams PowerShell Module version 3.0.0 and abve**

Reviewer: ```suggestion
**Updates in Teams PowerShell Module version 3.0.0 and above**
```

The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later (excluding updates mentioned previously for Teams PowerShell Module version 5.0.0):

> In the Teams PowerShell Module version 3.0.0 or later, filtering functionality is now limited to the following attributes (note that these changes are only rolled out in commercial environments at present and will be applicable to the latest TPM versions in GCCH and DOC environments starting March 31, 2023.):

Reviewer: ```suggestion
In the Teams PowerShell Module version 3.0.0 or later, filtering functionality is now limited to the following attributes (note that these changes are only rolled out in commercial environments including GCC at present, and will be applicable to the latest TPM versions in GCC High and DoD environments starting March 31, 2023.):
```

- UserValidationErrors                   
- WhenCreated    
     

> The updates below are now applicable to the output in scenarios where "-identity" parameter is not used:

Reviewer 2: ```suggestion
The following updates are now applicable to the output in scenarios where "-identity" parameter is not used:
```

- UserValidationErrors                   
- WhenCreated    
     
The updates below are now applicable to the output in scenarios where "-identity" parameter is not used:

- Only valid OnPrem users would be available in the output: These are users that are DirSyncEnabled and have a valid OnPremSipAddress or SIP address in ShadowProxyAddresses.
- Guest users are now available in the output.
- Unlicensed Users: Unlicensed users would show up in the output for 30 days post-license removal.
- Soft deleted users: These users will be displayed in the output of Get-CsOnlineUser and the TAC Manage Users page by default with SoftDeletionTimestamp set to a value.

If any information is required for a user that is not available in the output (when not using "-identity" parameter) then it can be obtained using the "-identity" parameter. Information for all users would be available using the "-identity" parameter until they are hard deleted.

> If Guest users and SoftDeletedUsers are not required in the output then they can be filtered out by using filter on AccountType and SoftDeletionTimestamp respectively.
> 
> ### -------------------------- Example --------------------------
> ```
> Get-csOnlineUser -Filter {AccountType -ne 'Guest'}
> Get-csOnlineUser -Filter {SoftDeletionTimestamp -eq $null}
> ```

Reviewer: ```suggestion
If Guest users and SoftDeletedUsers are not required in the output then they can be filtered out by using filter on AccountType and SoftDeletionTimestamp respectively. For example:
Get-csOnlineUser -Filter {AccountType -ne 'Guest'}
Get-csOnlineUser -Filter {SoftDeletionTimestamp -eq $null}
```

Reviewer: I don't think that example format is expected in this section.

**Updates in Teams PowerShell Module version 3.0.0 and above**

> The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later, excluding updates mentioned previously for Teams PowerShell Module version 5.0.0 (note that these changes are only rolled out in commercial environments at present and will be applicable to the latest TPM versions in GCCH and DOD environments starting March 31, 2023.):

Reviewer 2: ```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later, excluding updates mentioned previously for Teams PowerShell Module version 5.0.0 (note that these changes are only rolled out in commercial environments at present and will be applicable to the latest TPM versions in GCCH and DOD environments starting March 31, 2023.):
```

> The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later, excluding updates mentioned previously for Teams PowerShell Module version 5.0.0 (note that these changes are only rolled out in commercial environments at present and will be applicable to the latest TPM versions in GCCH and DOD environments starting March 31, 2023.):

Reviewer: ```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later, excluding updates mentioned previously for Teams PowerShell Module version 5.0.0 (note that these changes are only rolled out in commercial environments including GCC at present and will be applicable to the latest TPM versions in GCC High and DoD environments starting March 31, 2023.):
```
